remme/tp/node_account.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2018 REMME
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+
+import logging
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+
+from remme.protos.node_account_pb2 import (
```

Comment:
Please, follow the already presented style in the file, so migrate from

```
, ..
```

to

```

    .., 
    ..,
)
```

Also please, when you get some point, find all occurrences in the pull request and apply.

Reply:
Done.